src/lidar_subscriber.cpp

```diff
@@ -0,0 +1,156 @@
+
+#include <memory>
+#include <cmath>
+#include "rclcpp/rclcpp.hpp"
+
+#include "core-structs.h"
+
+
+#include "sensor_msgs/msg/laser_scan.hpp"
```

Comment:
Minor: please group your includes together. Headers with brackets should go together, and headers with quotes should go together.

Reply:
Done!